src/lib/discord.ts

```diff
@@ -1,7 +1,10 @@
-const sendMessage = (user: string, repoUrl : string) => {
+const sendMessage = (user: string, repoUrl: string) => {
   const request = (new XMLHttpRequest);
 
-  request.open("POST", "https://discord.com/api/webhooks/998629882764476448/ItO3_kUsGTKdDgqc6sTrraOmJFhIVTQmSbygV0z-dk-IjUXRB6H7MN5zakOB6ili7YTd");
+  request.open(
```

Comment:
comment this out please and change the discord url to `import.meta.env.DISCORD_WEBHOOK_URL` - the code being commented will not error out:

```
/*
 * request.open(
 *   "POST",
 *   import.meta.env.DISCORD_WEBHOOK_URL,
 * );
 */
```